webapp/components/description/description.js

```diff
@@ -158,9 +160,9 @@ Description.propTypes = {
 }
 
 const mapStateToProps = (state, props) => ({
-  ...state.user,
+  i18n: AppState.getI18n(state),
   content: R.pathOr('', ['descriptions', props.section, props.name, 'content'], state),
   editing: R.pathOr(false, ['descriptions', 'editing'], state)
 })
 
-export default connect(mapStateToProps, {fetchDescriptions, saveDescriptions, openEditor, closeEditor})(Description)
+export default connect(mapStateToProps, { fetchDescriptions, saveDescriptions, openEditor, closeEditor })(Description)
```

Comment:
Multiline?

Reply:
I'd address this in a more global refactoring